packages/botonic-plugin-flow-builder/src/index.ts

```diff
@@ -98,23 +98,19 @@ export default class BotonicPluginFlowBuilder implements Plugin {
 
   private updateRequestBeforeRoutes(request: PluginPreRequest) {
     if (request.input.payload) {
-      request.input.payload = this.removeSourceSeparatorFromPayload(
-        request.input.payload
-      )
+      request.input.payload = this.removeSourceSufix(request.input.payload)
 
       if (request.input.payload.startsWith(BOT_ACTION_PAYLOAD_PREFIX)) {
-        request.input.payload = this.updateBotActionPayload(
-          request.input.payload
-        )
+        request.input.payload = this.changePayload(request.input.payload)
       }
     }
   }
 
-  private removeSourceSeparatorFromPayload(payload: string): string {
+  private removeSourceSufix(payload: string): string {
     return payload.split(SOURCE_INFO_SEPARATOR)[0]
   }
 
-  private updateBotActionPayload(payload: string): string {
+  private changePayload(payload: string): string {
```

Comment:
i would name this with a more understandable name of what is doing exactly. I prefer the previous name or something that includes "replace". Also, would it make sense to have a specific class/set of functions to do extra logic for payloads?